webrtc/rtcpeerconnection/setRemoteDescription.html

```diff
@@ -0,0 +1,60 @@
+<!doctype html>
+<!--
+This test uses data only, and thus does not require fake media devices.
+-->
+
+<html>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  <title>RTCPeerConnection setRemoteDescription test</title>
+</head>
+<body>
+  <!-- These files are in place when executing on W3C. -->
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script type="text/javascript">
+  var test = async_test('Triggers onaddstream when called with a remote description.');
+
+  var gFirstConnection = null;
+
+  // This function starts the test.
+  test.step(function() {
+    gFirstConnection = new RTCPeerConnection(null);
+
+    gFirstConnection.onaddstream = test.step_func(function(event) {
```

Comment:
the real beauty is that it doesn't parse SDP at all to make these assertions :-)
We will need to agree on a way to parse SDP once I port some of the createOffer stuff but... future problem.